libs/shared/ui-components/src/Video/types.ts

```diff
@@ -0,0 +1,23 @@
+import { TImage, TRichText } from '@quansight/shared/types';
+
+type TSizeVariants = 'small' | 'medium' | 'large';
+type TAlignVariants = 'left' | 'right' | 'center';
+type TSpaceVariants = 'none' | 'small' | 'large';
+
+export type TVideoProps = {
+  url?: string;
+  thumbnail: TImage;
+  size: TSizeVariants;
+  align: TAlignVariants;
+  title: string;
+  sideText: TRichText;
+  text: TRichText;
```

Comment:
Could we add some documentation as comments? Maybe give a full-fledged example of a video object? For example, what is `sidetext` versus `text` and why are both required? Are there constraints on topSpace and bottomSpace taken together, for example should topSpace === bottomSpace?

Reply:
Speaking from the perspective of the desired design: no, topSpace and bottomSpace should be independently adjustable.

Reply:
I feel like my overall review comment got lost in the bit about constraints. That was meant to be an example to show that I think there needs to be a code comment (with examples) in the codebase for someone who has to later debug this code.

I'm asking for something like this:

```typescript
/**
 * - url: the URL where the video lives
 * - thumbnail: an object containing the alt text plus the name of the jpeg file
 *   in Storyblok that is a thumbnail for the video
 * - size: ?
 * - align: ?
 * - title: the title of the video
 * - sideText: ?
 * - text: ?
 * - topSpace: ?
 * - bottomSpace ?
 *
 * Are all combinations of possible values allowed? What constraints or other
 * information about this datatype might be useful to know here?
 *
 * One or two examples:
 *
 *   {
 *     url: '',
 *     thumbnail: {
 *     }
 *     ...
 *   }
 *   ...
 */
export type TVideoProps = {
```